src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -62,7 +62,7 @@ public abstract class BuildTestBase : IClassFixture<SharedBuildPerTestClassFixtu
         public static bool IsWorkloadWithMultiThreadingForDefaultFramework => s_buildEnv.IsWorkloadWithMultiThreadingForDefaultFramework;
         public static bool UseWebcil => s_buildEnv.UseWebcil;
         public static string GetNuGetConfigPathFor(string targetFramework)
-            => Path.Combine(BuildEnvironment.TestDataPath, targetFramework == "net10.0" ? "nuget10.config" : "nuget9.config");
+            => Path.Combine(BuildEnvironment.TestDataPath, targetFramework == DefaultTargetFramework ? $"nuget{Environment.Version.Major}.config" : $"nuget{Environment.Version.Major - 1}.config");
```

Comment:
I don't think we actually need separate nuget.config files, we just need to have all the `dotnet*` feeds so we could combine them into a single file and stop needing to update the filename too.